packages/smooth_app/lib/pages/product/edit_ingredients_page.dart

```diff
@@ -206,7 +199,7 @@ class _EditIngredientsPageState extends State<EditIngredientsPage> {
       ));
     }
 
-    return Scaffold(
+    final Scaffold scaffold = Scaffold(
```

Comment:
We started doing this a lot lately, I'm not a fan of having the scroll a lot to find the whole widget tree, what about putting it directly in the Consumer

Reply:
@M123-dev I agree with you: the reason behind that 2 step return is to avoid that `git` considers that all the text has changed just because the formatting added 2 spaces when we added a level with an extra `child`.
There were enough changes in this PR and I didn't want to add work to the code reviewers ;)